### PR TITLE
added myself

### PR DESCRIPTION
**Pull request by valcodes**
yep!

### CODE REVIEW

File: src/shared/december2017data.js [hunk 1]
```diff
@@ -2,40 +2,46 @@
 //  It should be updated by the people who attend the meetup and make a PR to update the file
 const december2017data = [
   {
```
Comment (ConnorBryan) [inline src/shared/december2017data.js]: @sedate13 
The styling should preferably remain the same, barring a discussion.

**Comment by m2mathew** [comment]
@sedate13 @ConnorBryan Yeah, let's keep the single quotes as the standard. Resolve the conflicts here, and I will merge. 😄

File: src/shared/december2017data.js (hunk 1 above)
Comment (m2mathew) [reply src/shared/december2017data.js]: Should probably be running a linter to avoid these types of issues. 🤔

**Comment by m2mathew** [comment]
Closing due to no response

**Pull request closed without merging**